### PR TITLE
chore: Update summary in kube-hunter

### PR DESCRIPTION
**Pull request by krol3**
Add summary section to the kube-hunter report component - https://github.com/aquasecurity/starboard-octant-plugin/issues/27.

image

### CODE REVIEW

File: pkg/plugin/view/kubehunter/report_view.go [hunk 1]
```diff
@@ -74,3 +80,25 @@ func createTable(section starboard.KubeHunterOutput) component.Component {
...
+func NewKubeHunterReportSummary(report *starboard.KubeHunterReport) (summary *component.Summary) {
+	totalHigh := 0
+	totalMedium := 0
+	totalLow := 0
+
+	for _, section := range report.Report.Vulnerabilities {
+		totalHigh += map[bool]int{true: 1, false: 0}[section.Severity == "high"]
+		totalMedium += map[bool]int{true: 1, false: 0}[section.Severity == "medium"]
+		totalLow += map[bool]int{true: 1, false: 0}[section.Severity == "low"]
+		println(section.Severity)
```
Comment (danielpacak) [inline pkg/plugin/view/kubehunter/report_view.go:93]: I think we don't need this print statement

File: pkg/plugin/view/kubehunter/report_view.go [hunk 2]
```diff
@@ -74,3 +80,25 @@ func createTable(section starboard.KubeHunterOutput) component.Component {
...
+	}
+
+	summary = component.NewSummary("Summary")
+
+	summary.Add([]component.SummarySection{
+		{Header: "High ", Content: component.NewText(strconv.Itoa(totalHigh))},
```
Comment (danielpacak) [inline pkg/plugin/view/kubehunter/report_view.go:99]: Let's keep header values lowercase like the severities that we summarize

File: pkg/plugin/view/kubehunter/report_view.go [hunk 3]
```diff
@@ -74,3 +80,25 @@ func createTable(section starboard.KubeHunterOutput) component.Component {
...
 	return table
 }
+
+func NewKubeHunterReportSummary(report *starboard.KubeHunterReport) (summary *component.Summary) {
+	totalHigh := 0
+	totalMedium := 0
+	totalLow := 0
+
+	for _, section := range report.Report.Vulnerabilities {
+		totalHigh += map[bool]int{true: 1, false: 0}[section.Severity == "high"]
```
Comment (danielpacak) [inline pkg/plugin/view/kubehunter/report_view.go:90]: I'd do that with switch statement to make this code a bit more readable and avoid creating maps in the loop

File: pkg/plugin/view/kubehunter/report_view.go (hunk 1 above)
Comment (krol3) [reply pkg/plugin/view/kubehunter/report_view.go:93]: my bad ! I forgot it

**Pull request merged**